src/Microsoft.Identity.Web.TokenAcquisition/DefaultTokenAcquisitionHost.cs

```diff
@@ -53,6 +56,15 @@ public MergedOptions GetOptions(string? authenticationScheme, out string effecti
 
             MergedOptions.UpdateMergedOptionsFromMicrosoftIdentityOptions(_microsoftIdentityOptionsMonitor.Get(effectiveAuthenticationScheme), mergedOptions);
```

Comment:
Same as for OWIN, we'll eventually remove the support for MicrosoftIdentityOptions when all works fine with MicrosoftAuthenticationOptions and SAL ...